### DISCUSSION TITLE
dicount code, response of dynamic field data

### DISCUSSION THREAD

**Discussion by Jocky92** [liedekef/events-made-easy#190] [Q&A] [answered]
is it possible to get a response of the dynamic field data?

i used "print_r($booking)" to show me the structure of the response array. this was really helpful but how do i now know how the array structure of the dynamic field data lookls like?

the dynamic filed data is not shown and loaded as long as i activate the "print_r($booking)" line.

**Comment by liedekef** [comment, marked as answer]
that is in the "answers" for the booking. See https://www.e-dynamics.be/wordpress/category/documentation/12-hooks-and-filters/ n search for eme_get_booking_post_answers and eme_get_booking_answers
